requirements/requirements-dev.txt

```diff
@@ -1,8 +1,7 @@
-#  spark-3.0.3-bin-hadoop2.7
```

Comment:
use "-" for branches (url)
snake "_" for python related file (convention)

Reply:
and it's good to have some comments to define for what purpose we have 2 requirements files, like
# Put development requirements here.